event_routing_backends/processors/xapi/event_transformers/forum_events.py

```diff
@@ -35,9 +40,31 @@ def get_object(self):
                 ),
             definition=ActivityDefinition(
                 type=constants.XAPI_ACTIVITY_DISCUSSION,
+                **kwargs
             )
         )
 
+    def get_context_activities(self):
+        context_activities = super().get_context_activities()
+
+        discussion = self.get_data('data.discussion.id')
+        if not discussion:
+            return context_activities
+
+        context_activities.grouping = [
```

Comment:
I'm using the grouping but we can also append it to the parent (which is the current course)

Reply:
I think that "grouping" is good 👍